prow/cluster/resources/monitoring/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: prometheus-prow-job-failure-rules
+  labels:
+    app: prometheus-operator
+{{ include "prometheus-operator.labels" . | indent 4 }}
+spec:
+  groups:
+  - name: ./prow.rules
+    rules:
+    - alert: KymaIntegration
```

Comment:
I think we should monitor all `postsubmit` and `periodics` jobs. Not only integration.